front/src/services/people/update.ts

```diff
@@ -41,8 +41,11 @@ const UPDATE_PERSON = gql`
   }
 `;
 
-export async function updatePerson(person: Person) {
-  const result = await apiClient.mutate({
+export async function updatePerson(
+  person: Person,
+  client: ApolloClient<NormalizedCacheObject> = apiClient,
```

Comment:
that signature is only for test purpose ? Maybe we should do a component test with the Apollo mock https://github.com/twentyhq/twenty/blob/main/front/src/pages/people/__stories__/People.stories.tsx and not do a unit test.
Or keep the unit test and do a jest.mock
```ts
jest.mock('./apollo', () => ({
  client: {
    // Your mocked myAPI object properties and methods go here
  }
}));
ts

Reply:
Good idea.